src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -100,7 +100,7 @@ const CheckAclPermsDocContainerForm = () => {
     >
       <form onSubmit={handleAclPermission} autoComplete="off">
         <div style={formRowStyle}>
-          <label htmlFor="set-acl-to">Check permissions to username's Documents container: </label>
+          <label htmlFor="set-acl-to">Check permissions to username&apos;s Documents container: </label>
```

Comment:
The linter recommends escaping apostrophe characters like this.

Reply:
While I'm not really a fan of using HTML entities for special characters (I think we've previously turn off linting rules for that), I'm not sure of a work around for that.

Think this would have to do.

Reply:
Yeah I have mixed feelings about it too. Although I get what they're going for. If you have these regularly (especially > and <) eventually one of them will break something. Most companies avoid this rule because they display strings like this in a separate document that's then imported as a JS constant. They do this so they can more easily translate the string. But we shouldn't worry about implementing something like that in PASS right now.

I can disable that rule again, or try to add an option to allow apostrophes.

Reply:
I think we can leave this rule on.